spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -665,7 +665,7 @@ def _write_on_machine_data_spec(
 
         n_bytes = (
             SYNAPSES_BASE_GENERATOR_SDRAM_USAGE_IN_BYTES +
-            (self.__n_synapse_types * BYTES_PER_WORD))
+            (self.__n_synapse_types * 2 * BYTES_PER_WORD))
```

Comment:
magic number!

Reply:
This is a magic number at the moment because I didn't know what to call it (it's from the branch where I switched from using S1615 to U3232 for the weight scale value, thus doubling this part of the n_bytes calculation).  If you have any ideas then please speak up...

Reply:
theres a dsg size thingy for data types. is there any way to get to the right one here for the correct types?

Reply:
S3231.size()

Reply:
Fixed